app/models/alchemy/page/page_scopes.rb

```diff
@@ -100,5 +96,35 @@ module Page::PageScopes
       #
       scope :sitemap, -> { from_current_site.published.contentpages.where(sitemap: true) }
     end
+
+    module ClassMethods
+
+      # All public pages
+      #
+      def published
+        publication_scope([
+          "#{table_name}.public_on <= ? AND".freeze,
+          "(#{table_name}.public_until IS NULL".freeze,
+          "OR #{table_name}.public_until >= ?)".freeze
+        ])
+      end
+
+      # All not public pages
+      #
+      def not_public
+        publication_scope([
+          "#{table_name}.public_on IS NULL OR".freeze,
+          "#{table_name}.public_on >= ? OR".freeze,
+          "#{table_name}.public_until <= ?".freeze
+        ])
```

Comment:
Style/IndentArray: Indent the right bracket the same as the first position after the preceding left parenthesis.